docs/pages/guides/building-for-tv.mdx

```diff
@@ -0,0 +1,222 @@
+
+- The environment variable `EXPO_TV` is set to `1`
+- The plugin parameter `isTV` is set to `true`
+
+Optionally, set the `showVerboseWarnings` property to `true` to see additional information during prebuild.
+
+```json app.json
+{
+  "plugins": [
+    [
+      "@react-native-tvos/config-tv",
+      {
+        "showVerboseWarnings": true
+      }
+    ]
+  ]
+}
+```
+
+</Step>
+
+<Step label="3">
+
+## Run prebuild
+
+Set the `EXPO_TV` environment variable, and run prebuild to make the TV modifications to the project.
+
+<Terminal cmd={['export EXPO_TV=1', 'npx expo prebuild --clean']} />
+
+> **Note**: The `--clean` argument is recommended, and is required if you have existing Android and iOS directories in the project.
+
+</Step>
+
+<Step label="4">
+
+## Build for Apple TV
+
+After running prebuild, run the following command to build and run the app on an Apple TV simulator:
+
+<Terminal
+  cmd={[
+    'npx expo run:ios'
+  ]}
+/>
+
+</Step>
+
+<Step label="5">
+
+## Build for Android TV
+
+After running prebuild, start an Android TV emulator, and then use the same command as you would to start the app on an Android phone emulator.
+
+<Terminal
+  cmd={[
+    'npx expo run:android'
+  ]}
+/>
+
+</Step>
+
+<Step label="6">
+
+## Revert TV changes and build for phone
+
+You can revert the changes for TV and go back to phone development by unsetting `EXPO_TV` and running prebuild again:
+
+<Terminal cmd={['unset EXPO_TV', 'npx expo prebuild --clean']} />
+
+</Step>
+
+<Step label="7">
+
+## Create EAS build profiles for both TV and phone
+
+Since the TV build is driven by the value of an environment variable, it is easy to set up EAS Build profiles that build from the same source but target TV instead of phone.
+
+The following example **eas.json** shows how to extend existing profiles (`development` and `preview`) to create TV profiles (`development_tv` and `preview_tv`).
+
+```json eas.json
+{
+  "cli": {
+    "version": ">= 5.2.0"
+  },
+  "build": {
+    "base": {
+      "distribution": "internal",
+      "ios": {
+        "simulator": true
+      },
+      "android": {
+        "buildType": "apk",
+        "withoutCredentials": true
+      },
+      "channel": "base"
+    },
+    "development": {
+      "extends": "base",
+      "android": {
+        "gradleCommand": ":app:assembleDebug"
+      },
+      "ios": {
+        "buildConfiguration": "Debug"
+      },
+      "channel": "development"
+    },
+    "development_tv": {
+      "extends": "development",
+      "env": {
+        "EXPO_TV": "1"
+      },
+      "channel": "development"
+    },
+    "preview": {
+      "extends": "base",
+      "channel": "preview"
+    },
+    "preview_tv": {
+      "extends": "preview",
+      "env": {
+        "EXPO_TV": "1"
+      },
+      "channel": "preview"
+    }
+  },
+  "submit": {}
+}
+```
+
+</Step>
+
+## Learn more
+
+For a complete example, refer to the [douglowder/IgniteTV repository](https://github.com/douglowder/IgniteTV).
```

Comment:
```suggestion
<BoxLink
  title="IgniteTV"
  description="A complete example is available at GitHub."
  href="https://github.com/douglowder/IgniteTV"
  Icon={GithubIcon}
/>
```

Let's use `BoxLink` to refer to the GitHub example. Also imported `BoxLink` and `GithubIcon` at the top of this file.